Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
+    yum clean all && /usr/sbin/sshd-keygen
+
+RUN mkdir /var/run/sshd
+#RUN sed -i 's/LogLevel INFO/LogLevel VERBOSE/' /etc/ssh/sshd_config
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+
+# gems
+#ENV GEM_HOME="/usr/local/lib/ruby/gems/2.1.0"
+RUN ["bash", "-c", "gem install --no-ri --no-rdoc bunny"]
+
+# git user
+RUN groupadd -r git &&\
+     useradd -r -s /bin/bash -g git git && \
+     mkdir --parent /home/git && \
+     chown -R git:git /home/git
+
+
+# gitlab-shell setup
+COPY . /home/git/gitlab-shell
+WORKDIR /home/git/gitlab-shell
+RUN ["bash", "-c", "./bin/install"]
+
+COPY authorized_keys /home/git/.ssh/authorized_keys
+COPY dummy-redis-cli.sh /usr/bin/redis-cli
```

Comment:
Since this is configured in the config file, I don't think we need to `COPY` it here. We should be able to just set the PATH to it in the config file, since there's already a copy of it in `/home/git/gitlab-shell`. Actually, I think I'll move it to `./bin` here in the repo and update the config file accordingly. WDYT?